apps/vm_minimal/CMakeLists.txt

```diff
@@ -75,6 +75,21 @@ elseif("${KernelARMPlatform}" STREQUAL "tx2")
     AddToFileServer("linux" "${CMAKE_SOURCE_DIR}/projects/camkes-vm-images/tx2/linux")
     AddToFileServer("linux-dtb" "${CMAKE_SOURCE_DIR}/projects/camkes-vm-images/tx2/linux-dtb")
 
+elseif("${KernelARMPlatform}" STREQUAL "zynqmp")
+    AddToFileServer("linux" "${CMAKE_SOURCE_DIR}/projects/camkes-vm-images/zynqmp/linux")
+
+    set(device_tree_src "${CMAKE_SOURCE_DIR}/projects/camkes-vm-images/zynqmp/linux.dts")
+
+    add_custom_command(
```

Comment:
I noticed this command needs `DEPENDS ${device_tree_src}` in order to rebuild linux-dtb if the dts is updated